File: scratch_query.py
import sqlite3
import json

try:
    conn = sqlite3.connect('C:\\Users\\Raman Duggal\\OneDrive\\BIZSUITE\\season\\2\\jrmd.sqlite')
    cursor = conn.cursor()
    print("Database opened successfully.")
    
    # Get tables
    cursor.execute("SELECT name FROM sqlite_master WHERE type='table'")
    tables = [r[0] for r in cursor.fetchall()]
    print("Tables:", tables)
    
    for t in ['customer_payments', 'payments', 'transactions']:
        if t in tables:
            try:
                cursor.execute(f"SELECT id, data FROM {t}")
                rows = cursor.fetchall()
                print(f"Checking table {t}, found {len(rows)} rows...")
                for row_id, data_str in rows:
                    data = json.loads(data_str)
                    if data.get('id') == 'L00433' or data.get('paymentId') == 'L00433':
                        print(f"FOUND in {t}:", json.dumps(data, indent=2))
            except Exception as e:
                print(f"Error checking table {t}: {e}")
                
except Exception as e:
    print("Error:", e)

File: scratch_query.js
const Database = require('better-sqlite3');
const path = require('path');

try {
  const db = new Database('C:\\Users\\Raman Duggal\\OneDrive\\BIZSUITE\\season\\2\\jrmd.sqlite', {readonly: true});
  console.log("Database opened successfully.");
  
  // Check customer_payments table
  try {
    const rows = db.prepare("SELECT name FROM sqlite_master WHERE type='table'").all();
    console.log("Tables:", rows.map(r => r.name));
    
    // Search customer_payments and payments tables
    const tables = ['customer_payments', 'payments', 'transactions'];
    for (const t of tables) {
      try {
        const results = db.prepare(`SELECT id, data FROM ${t}`).all();
        console.log(`Checking table ${t}, found ${results.length} rows...`);
        for (const row of results) {
          const data = JSON.parse(row.data);
          if (data.id === 'L00433' || data.paymentId === 'L00433') {
            console.log(`FOUND in ${t}:`, JSON.stringify(data, null, 2));
          }
        }
      } catch(e) {
        console.log(`Error checking table ${t}:`, e.message);
      }
    }
  } catch(e) {
    console.log("Query Error:", e.message);
  }
} catch(e) {
  console.log("DB Open Error:", e);
}
